components/automate-gateway/api/iam/v2/users.proto

```diff
@@ -33,7 +85,21 @@ service Users {
     option (chef.automate.api.iam.policy).resource = "iam:users";
     option (chef.automate.api.iam.policy).action = "iam:users:list";
   };
+
+  /*
+  Get a user
+
+  Returns the details for a user.
```

Comment:
not sure if we really need `local` before user every time but maybe just for consistency? 🤷‍♀